acquisitions/src/main/resources/domain/mod-finance/finance-junit.feature

```diff
@@ -0,0 +1,28 @@
+Feature: mod-finance integration tests
+
+  Background:
+    * url baseUrl
+    * table modules
+      | name                  |
+      | 'mod-finance'         |
+      | 'mod-login'           |
+      | 'mod-permissions'     |
+      | 'mod-finance-storage' |
+      | 'mod-configuration'   |
+
+    * def testAdmin = {tenant: '#(testTenant)', name: 'test-admin', password: 'admin'}
```

Comment:
remove this
 * def testAdmin = {tenant: '#(testTenant)', name: 'test-admin', password: 'admin'}
    * def testUser = {tenant: '#(testTenant)', name: 'test-user', password: 'test'}

Reply:
Done